### DISCUSSION TITLE
Cannot find module 'client-only' after deploying prebuilt

### DISCUSSION THREAD

**Discussion by owboateng** [vercel/community#4387] [Help]
### Summary

After deploying to vercel, I get an internal server error when I try to open a dynamic page.
Cannot find module 'client-only'
Require stack:
- /var/task/node_modules/styled-jsx/dist/index/index.js
- /var/task/node_modules/styled-jsx/index.js
- /var/task/node_modules/next/dist/server/render.js
- /var/task/node_modules/next/dist/server/next-server.js
- /var/task/___next_launcher.cjs
Did you forget to add it to "dependencies" in `package.json`?

### Example

https://tslearn-jplxv6rfh-owboateng.vercel.app/blog/view/MH3hVYVLlvbBpSx8UADJB/a-lesson-in-democracy

### Steps to Reproduce

OS: MacOS Ventura

% vercel --version                        
Vercel CLI 32.4.1
32.4.1

vercel build && vercel deploy --prebuilt

Visit site and try open a dynamic page: blog/view/MH3hVYVLlvbBpSx8UADJB/a-lesson-in-democracy

**Comment by andrewkmin** [comment]
@owboateng did you ever resolve this issue?
